dll/win32/msgina/shutdown.c

```diff
@@ -336,6 +348,17 @@ ShutdownOnInit(
             SendMessageW(hwndList, CB_SETITEMDATA, idx, WLX_SAS_ACTION_SHUTDOWN_REBOOT);
     }
 
+    /* Restart in NT Native Mode */
+    if (pContext->ShutdownOptions & WLX_SHUTDOWN_STATE_NT_MODE)
```

Comment:
This also responds to reboot and logoff now (see bitflag definition)

Reply:
Damn! Rookie mistake right there. Fixing now.